index.html

```diff
@@ -21,6 +21,10 @@ <h2>Experience</h2>
       <li>linux </li>
     </ul>
   </section>
+  <section id="skills">
+    <h2>"Skills"</h2>
```

Comment:
You don't need the `"` quotation marks around Skills; since this is HTML, everything is interpreted as a string already.